doc/en/how-to/fixtures.rst

```diff
@@ -735,8 +735,81 @@ does offer some nuances for when you're in a pinch.
 .. code-block:: pytest
 
    $ pytest -q test_emaillib.py
-   .                                                                    [100%]
-   1 passed in 0.12s
+  .                                                                    [100%]
+  1 passed in 0.12s
+
+Note on finalizer order
+""""""""""""""""""""""""
+
+Finalizers are executed in a first-in-last-out order.
+For yield fixtures, the first fixture to run is the right-most one, i.e. the last test parameter.
```

Comment:
```suggestion
For yield fixtures, the first teardown code to run is from the right-most fixture, i.e. the last test parameter.
```

The fixture setup code is executed left-to-right, while the teardown code is right-to-left.

Reply:
That's right, thanks for clarifying :)